test/karma.conf.js

```diff
@@ -23,7 +25,7 @@ module.exports = function(config) {
 
   config.set({
     basePath: '..',
-    frameworks: ['qunit', 'detectBrowsers'],
+    frameworks: ['qunit'],
```

Comment:
why was detectBrowsers removed? This is still useful for testing locally

Reply:
Because it picks up Canary and potentially browsers that the tests don't work in, which can result in false positives. Maybe there is a better solution, but false positives can result in a lot of lost time.

Reply:
Though I understand your frustration, I don't believe reducing tested browsers helps us in the long run. Having the tests run on more browsers is a good way to ensure browsers we don't normally develop in are tested. 

An alternative to what you did here would be to remove browsers that we know the tests don't run in from the list of browsers that were detected. I'll be happy to dig up an example of how to do that if needed

Reply:
It's not an emotional choice, it's pragmatism: if everyone is running the tests differently, our results are not scientific. If we want to run the tests on more platforms, I would suggest we add them explicitly rather than saying "whatever happens to be installed"

Reply:
Additionally, any browsers we add should be confirmed to work with the tests before committing.

Reply:
the tests will then fall for contributors that don't have those explicit browsers installed

Reply:
That seems like a good behavior if your system can't run the full test suite.

Reply:
After discussion we're changing it to detectBrowsers with a whitelist.